src/Popup.tsx

```diff
@@ -44,37 +60,50 @@ const Popup = ({
   const [computedPosition, setComputedPosition] = useState<
     PositionType | string
   >(position);
+  const isMounted = useIsMounted();
 
   const handleAnchorLayout = useCallback(() => {
     if (anchorRef.current) {
-      anchorRef.current.measureInWindow((x, y, width, height) => {
-        setAnchorLayout({
-          x,
-          y,
-          width,
-          height,
-        });
-      });
+      const timer = setTimeout(() => {
```

Comment:
Despite adding this timeout, there are still some layout issues with the popover content on web. When opening the popup, it will jump around until it has finished calculating the optimal dimensions for the content. A bigger fix for that might be rendering the content transparently in the center of the screen and then placing/showing it after the content size is known.